docs_user/modules/proc_deploying-a-ceph-ingress-daemon.adoc

```diff
@@ -103,12 +103,8 @@ spec:
   virtual_interface_networks:
     - <external_network>
   ssl_cert: |
```

Comment:
can we put this key in the form: `ssl_cert: ...` like done for the other sample?